test/Interop/Cxx/class/synthesized-initializers.swift

```diff
@@ -0,0 +1,38 @@
+// RUN: %target-swift-frontend -I %S/Inputs -enable-cxx-interop -emit-sil %s | %FileCheck %s
```

Comment:
Thanks! Could you change it to an `-emit-silgen` test and CHECK the instructions in the actual initializer before inlining? That'd be more precise, less noisy in CHECK lines, and won't rely on the behavior of the inliner.